firewall/iptables/iptables.go

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright (C) 2018 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package iptables
+
+import (
+	"bufio"
+	"bytes"
+	"os/exec"
+	"strings"
+
+	log "github.com/cihub/seelog"
+	"github.com/mysteriumnetwork/node/firewall"
+	"github.com/pkg/errors"
+)
+
+const (
+	logPrefix = "[iptables] "
+
+	outputChain     = "OUTPUT"
+	killswitchChain = "CONSUMER_KILL_SWITCH"
+
+	addChain         = "-N"
+	appendRule       = "-A"
+	insertRule       = "-I"
+	listRules        = "-S"
+	removeRule       = "-D"
+	removeChainRules = "-F"
+	removeChain      = "-X"
+
+	jumpTo        = "-j"
+	sourceIP      = "-s"
+	destinationIP = "-d"
+	module        = "-m"
+
+	protocol        = "-p"
+	tcp             = "tcp"
+	udp             = "udp"
+	destinationPort = "--dport"
+
+	conntrack    = "conntrack"
+	ct_state     = "--ctstate"
+	ct_state_new = "NEW"
+
+	reject = "REJECT"
+	accept = "ACCEPT"
+
+	version = "--version"
+)
+
+var iptablesExec = func(args ...string) ([]string, error) {
+	args = append([]string{"/sbin/iptables"}, args...)
```

Comment:
there is no such wrapper. What do you mean?

Reply:
Ok it seems that on docker we don't need sudo at all. On debian package /sbin/iptables are already on sudoers list. So "sudo" can safely be removed

Reply:
@zolia You implemented `sudoers` feature. It will not allow this command in .deb package, correct?

Reply:
@Waldz even if you list command in sudoers file you still need to prefix it with sudo prefix.. you just don't need a root password anymore.

Reply:
we created `bin/package/config/linux/nonpriv-ip` because openvpn does expect a single script as  a substitution for ip routing functionality executable. There is no point to create more wrappers here if we can wrap it directly in a go code.

Reply:
A side note. /sbin/iptables where used previously in code (for masquarading from provider site) in exactly the same way. I don't see any reason why it should work now (btw it works - tested on docker and linux machines).